src/modules/rewards/components/RewardsActivitiesListItem.tsx

```diff
@@ -167,84 +187,78 @@ const RewardsActivitiesListItem: FC<RewardActivitiesListItemProps> = ({ userId,
             )}
 
             {usersSingleActivity?.status === 'PENDING' && (
-              <Box display="flex" alignItems={{ ml: 'flex-start', initial: 'center' }} flexDirection="column" minWidth="100px">
+              <Box
+                display="flex"
+                alignItems={{ ml: 'flex-start', initial: 'center' }}
+                flexDirection="column"
+                minWidth="100px"
+              >
                 <Skeleton width="100%" isLoading={isLoading}>
                   <Button
                     variant="tertiary"
                     size="small"
                     css={css`
-                 width: 100%;
-               `}
+                      width: 100%;
+                    `}
                     disabled={true}
                   >
-                    Pending Verification
+                    Pending
                   </Button>
                 </Skeleton>
               </Box>
             )}
 
-            {usersSingleActivity && usersSingleActivity?.status !== 'COMPLETED' && (
-              <Box display="flex">
-
-                <ActivityButton
-                  userId={userId}
-                  activityTypeId={activity.id}
-                  activityType={activity.activityType}
-                  refetchActivity={refetchActivity}
-                  setErrorMessage={setErrorMessage}
-                />
-
-              </Box>
-            )}
+            {usersSingleActivity &&
+              usersSingleActivity?.status !== 'COMPLETED' &&
+              usersSingleActivity?.status !== 'PENDING' && (
+                <Box display="flex">
+                  <ActivityButton
+                    userId={userId}
+                    activityTypeId={activity.id}
+                    activityType={activity.activityType}
+                    refetchActivity={refetchActivity}
+                    setErrorMessage={setErrorMessage}
+                  />
+                </Box>
+              )}
           </Box>
```

Comment:
@abhishek-01k we discussed this already multiple times.

If we have a Activity Button component then why are we having 2 separate button in RewardsActivityListItem for claimed and pending and then a third button component ActivityButton separately.

This was supposed to be a single button component handling everything inside it.